wherehows-web/app/routes/search.js

```diff
@@ -3,7 +3,7 @@ import { isBlank } from '@ember/utils';
 import $ from 'jquery';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import buildUrl from 'wherehows-web/utils/build-url';
-import highlightResults from 'wherehows-web/utils/api/datasets/highlight-results';
+import createSearchEntries from 'wherehows-web/utils/datasets/create-search-entries';
```

Comment:
@cptran777 I'm still not sure what the intent of this function is, the original implementation was to highlight the keywords in the search result entries with bold tags, this doesn't seem to do anything of that sort? Is that the case?